dev-proxy/ProxyHost.cs

```diff
@@ -396,6 +397,13 @@ private static Command CreateCertEnsureCommand(ILogger logger)
         return certEnsureCommand;
     }
 
+    private static Command CreateCertRemoveCommand(ILogger logger)
```

Comment:
Since this command is removing resources, let's ask for confirmation by default before executing its logic. Users can suppress the prompt by using the `--force` switch.

Reply:
To clarify, should we add some confirmation prompt in console and bypass it with `--force` option? 
If we are going to control the appearance of the confirmation dialogue to remove Root certificate... I have to find out if it is feasible.

Reply:
Yes, when the command is run without `--force`, we should prompt for confirmation, ie. `Do you want to remove the root certificate (y/N)`, `No` being the default. When the command is ran with `--force` we don't show the prompt and proceed with removing the certificate. The prompt should only control if we run the command or exit early. We shouldn't suppress underlying prompts or dialogs.